spark-screenshot-testing/src/test/kotlin/com/adevinta/spark/icons/AnimatedIconsScreenshot.kt

```diff
@@ -40,44 +40,144 @@ internal class AnimatedIconsScreenshot {
     @get:Rule
     val paparazzi = paparazziRule()
 
+//    Blocked until https://github.com/cashapp/paparazzi/pull/1645 is merged and available
```

Comment:
Can you please uncomment and add a proper `@Ignore("Blocked until https://github.com/cashapp/paparazzi/pull/1645 is merged and available")`